src/CommentSold/Api/Resources/Request/AbstractRequest.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CommentSold\Api\Resources\Request;
+
+use CommentSold\Api\Resources\AbstractResource;
+
+abstract class AbstractRequest extends AbstractResource
+{
+}
```

Comment:
This doesn't seem necessary at all. You could change it to an interface if you want to make sure the right things are being passed around